docs/DeveloperGuide.adoc

```diff
@@ -233,12 +232,7 @@ If the user input is incorrect due to any of the reasons above, the usage syntax
 
 We will demonstrate how a `**Transaction**` is added into `**THRIFT**` and how the back-end handles each step of the process:
 
-Step 1. The user launches the application for the first time. `**THRIFT**` does not contain any other `**Transaction**` and the user is
-about to add their own. An empty transaction list is initialized.
-
-image::add-transaction/AddTransaction0.png[]
-
-Step 2. The user executes **add_expense n/Laksa v/3.50** to insert an `**Expense**` with `**Description**` set to "Laksa"
+Step 1. The user executes **add_expense n/Laksa v/3.50** to insert an `**Expense**` with `**Description**` set to "Laksa"
```

Comment:
... with **its** \`\*\*Description\*\*\` set to ...

... and **its** \`\*\*Value\*\*\` set to ...